presto-main/etc/config.properties

```diff
@@ -46,7 +46,8 @@ plugin.bundles=\
   ../presto-postgresql/pom.xml, \
   ../presto-thrift/pom.xml, \
   ../presto-tpcds/pom.xml, \
-  ../presto-google-sheets/pom.xml
+  ../presto-google-sheets/pom.xml, \
+  ../presto-snowflake/pom.xml
```

Comment:
There is no need to add every connector to this property.

Reply:
Removed.